intermine_builder/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ -d ${MINE_NAME:-biotestmine} ] && [ ! -z "$(ls -A ${MINE_NAME:-biotestmine})" ]; then
+if [ -d ${MINE_NAME:-biotestmine} ] && [ ! -z "$(ls -A ${MINE_NAME:-biotestmine})" ] && [ -z ${IM_DATA_FROM_PATH} ]; then
```

Comment:
Because of the change I requested in https://github.com/intermine/intermine_boot/pull/37, you should use `FORCE_MINE_BUILD` instead of `IM_DATA_FROM_PATH`.

Reply:
Yes, would do it.

Reply:
done.